src/main/java/pages/SiteVisitPage.java

```diff
@@ -48,4 +53,40 @@ public boolean isBackButtonAvailable() {
         }
         return status;
     }
+
+    public boolean isCreateTestButtonDisplayed() {
+        return findElementById(CREATE_TEST_ID).isDisplayed();
+    }
+
+    public boolean isEndVisitButtonDisplayed() {
+        return findElementById(END_VISIT_ID).isDisplayed();
+    }
+
+    public boolean isAtfRowDisplayed(String atfName) {
+        return findElementByXpath("//XCUIElementTypeButton[contains(@name,'" + atfName + " Started site visit')]").isDisplayed();
+    }
+
+    public boolean isCurrentDateDisplayed() {
+        LocalDateTime ldt = LocalDateTime.now();
```

Comment:
We need to have time validation based on timezone in browserstack CI, Browserstack local by using browserstack.timezone property to set a specific timezone for device, and make sure that the validation considers this and can handle local simulator, local real device, local browserstack, Ci browserstack

Reply:
done